digitalstrategie/templates/base.html

```diff
@@ -33,6 +33,7 @@
         <script src="//www.berlin.de/i9f/r1/bundle/shariff.min.js"></script>
         <script src="//www.berlin.de/i9f/v4/js/bo-foot-rebrush.js"></script>
 
+        <script type="text/javascript" src="{% static 'video.js' %}"></script>
```

Comment:
think you can omit the type, as it is javascript (according to MDN).
But really nit-picky so up to you, feel free to merge otherwise.

Reply:
it's more in line with the js we have right above, so will change